tools/editorconfig/.editorconfig

```diff
@@ -0,0 +1,20 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+charset = utf-8
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.{yml,twig,php}]
+indent_size = 4
```

Comment:
@Soullivaneuh @greg0ire The most used convention nowadays, is 2 spaces for js/css, but maybe being consistent with php files is better ?